nym-vpn-core/crates/nym-gateway-probe/src/types.rs

```diff
@@ -11,6 +11,16 @@ pub struct ProbeResult {
 pub struct ProbeOutcome {
     pub as_entry: Entry,
     pub as_exit: Option<Exit>,
+    pub wg: Option<Wg>,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct Wg {
```

Comment:
Maybe something along the lines of  `ProbeResult`, `ProbeStatus`, `ProbeReport`?